### PR TITLE
TEST 200 Positions admin

### PR DESCRIPTION
**Pull request by kristiyanplpeev**
Front end - added positions view, where an admin can see all the positions with their coefficients and also to change each coefficient.

Back end - added coefficient and sort order columns in position table with default values for them, also added endpoint for updating position coefficient.

### CODE REVIEW

File: client/src/components/Positions/Positions.tsx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+          </Button>
+          <Button
+            variant="outlined"
+            color="error"
+            className="change-position-modal-buttons"
+            onClick={() =>
+              this.setState({
+                openModal: false,
+              })
+            }
```
Comment (stoyanov-ivelin) [inline client/src/components/Positions/Positions.tsx:151]: You could have a handleClose method for this logic

**Review by stoyanov-ivelin — CHANGES_REQUESTED** [review]
Great job! Left some minor comments.

File: client/src/App.tsx [hunk 2]
```diff
@@ -38,11 +39,13 @@ interface AppState {
 }
 
 class App extends Component<AppProps, AppState> {
+  private userState: IUserState;
```
[the remark this answers is not in the document]
Comment (kristiyanplpeev) [reply client/src/App.tsx:42]: done.

File: client/src/components/Positions/Positions.tsx (hunk 1 above)
Comment (kristiyanplpeev) [reply client/src/components/Positions/Positions.tsx:151]: done.

File: client/src/services/UserService.ts [hunk 3]
```diff
@@ -71,6 +71,13 @@ class UserService implements IUserService {
     };
     await this.restClient.put(`users/roles`, { data });
   };
+  updatePositionCoefficient = async (positionId: string, newCoefficient: number): Promise<void> => {
```
[the remark this answers is not in the document]
Comment (kristiyanplpeev) [reply client/src/services/UserService.ts:74]: done.

**Review by stoyanov-ivelin — APPROVED** [review]
LGTM!

File: client/src/components/Positions/Positions.tsx [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    });
+  };
+
+  handleConfirmClick = async (): Promise<void> => {
+    try {
+      this.setState({
+        loading: true,
+      });
+      await this.userService.updatePositionCoefficient(this.state.expandedPosition, +this.state.expandedInput.value);
+      const updatedPositions = this.state.positions.map((position) => {
```
Comment (MilenIvanovv) [inline client/src/components/Positions/Positions.tsx:175]: There is no problem here but I would've got all the positions' coefficients from the back-end. There will be no performance issue, easier to sync (update the state), and usually in the browser often the data may not be up to date.

**Review by MilenIvanovv — APPROVED** [review]
LGTM

File: client/src/components/Positions/Positions.tsx (hunk 4 above)
Comment (kristiyanplpeev) [reply client/src/components/Positions/Positions.tsx:175]: done.

**Pull request merged**